server.js

```diff
@@ -56,34 +58,32 @@ if (localConfigExists) {
   }
 }
 
-new WebpackDevServer(webpack(config), serverConfig).listen(
-  port,
-  'localhost',
-  function(err) {
-    if (err) {
-      console.log(err);
-    }
-    const barAscii =
-      '------------------------------------------------------------------------------------------';
+new WebpackDevServer(webpack(config), serverConfig).listen(port, host, function(
+  err
+) {
+  if (err) {
+    console.log(err);
+  }
+  const barAscii =
+    '------------------------------------------------------------------------------------------';
 
-    console.log(barAscii);
-    console.log(`> perf.html is available at: http://localhost:${port}\n`);
-    if (port === 4242) {
-      console.log(
-        '> You can change this default port with the environment variable PERFHTML_PORT.\n'
-      );
-    }
-    if (localConfigExists) {
-      console.log(
-        '> We used your local file "webpack.local-config.js" to mutate webpack’s config values.'
-      );
-    } else {
-      console.log(stripIndent`
+  console.log(barAscii);
+  console.log(`> perf.html is available at: http://${host}:${port}\n`);
```

Comment:
I still think this is a bit incorrect if `host` is `0.0.0.0` like you advise. Maybe this change would make it OK:
```suggestion
  console.log(`> perf.html is listening at: http://${host}:${port}\n`);
```
What do you think?